README.md

```diff
@@ -22,16 +42,26 @@ ts_m2fpga
 │   ├── ilcCommmunication.md
 ├── fpgaInterface
 │   ├── NiFpga_mainFPGA.h
-│   ├── NiFpga_mainFPGA.lvbitx
 │   ├── NiFpga.c
 │   ├── NiFpga.h
+│   ├── NiFpga_mainFPGA.lvbitx
+│   ├── bitfile
+│       ├── m2fpga_FPGATarget_mainFPGA_<random_characters>.lvbitx
 ├── README.md
 └── .gitignore
 ```
 
 - `src`: contains the LabVIEW FPGA project.
 - `doc`: documentation like `versionHistory.md`.
 - `fpgaInterface`: in here are all files generated by the FPGA Interface C API.
+- `bitfile`: contains the bitfile (.lvbitx).
+The bitfile contains all the configuration information from the LabVIEW block diagram defining the internal logic and digital circuit of the FPGA and device-specific information from other files associated with the FPGA target.
+When the FPGA application executes, the bitfile loads on the FPGA chip and reconfigures the gate array logic.
+Follow [this link](https://knowledge.ni.com/KnowledgeArticleDetails?id=kA03q000000YHVTCA4&l=en-US) if you want to know more about the LabVIEW FPGA compilation process.
+
+Random characters in the `lvbitx` file name appear when the **mainFPGA.vi** is compiled.
```

Comment:
I do not understand this sentence at all: "Random characters in the `lvbitx` file name".

Reply:
I realized there is not necessary to explain these random characters, because if the developer compiles the FPGA again, the bitfile name will be the same.